src/__stories__/stacking-group-story.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import {skinVars} from '..';
+import {ThemeVariant} from '../theme-variant-context';
+import StackingGroup from '../stacking-group';
+
+export default {
+    title: 'Components/StackingGroup',
+    argTypes: {
+        size: {
+            options: [24, 32, 64],
+            control: {type: 'select'},
+        },
+        type: {
+            options: ['avatar', 'image'],
+            control: {type: 'select'},
+        },
+    },
+    parameters: {
+        fullScreen: true,
+    },
+};
+
+type Args = {
+    size: 24 | 32 | 64;
+    data: object[];
+    type: 'avatar' | 'image';
+    stacked?: boolean;
+    inverse: boolean;
+};
+
+export const Default: StoryComponent<Args> = ({size = 24, data, type, stacked, inverse}) => {
+    return (
+        <ThemeVariant isInverse={inverse}>
+            <div
+                style={{
+                    padding: 16,
+                    width: 'fit-content',
+                    background: inverse ? skinVars.colors.backgroundBrand : skinVars.colors.background,
+                    lineHeight: 0,
+                }}
+                data-testid="stacking-group"
+            >
+                <StackingGroup size={size} type={type} stacked={stacked} data={data} />
+            </div>
+        </ThemeVariant>
+    );
+};
+
+Default.storyName = 'Stacking Group';
+Default.args = {
+    size: 24,
+    data: [
```

Comment:
Instead of harcoding here an array, I'd include a slider with the number of items to include.
Also include an arg for `maxItems` prop, please